internal/lib/constants.go

```diff
@@ -172,6 +172,7 @@ const (
 	IPAMProviderInfoblox                       = "IPAMDNS_TYPE_INFOBLOX"
 	IPAMProviderCustom                         = "IPAMDNS_TYPE_CUSTOM"
 	SharedVipServiceKey                        = "SharedVipService"
+	HttpRulePkiAndDestCASetErr                 = "PKIProfile and DestinationCA has been set in a HTTPRule. Only one of the field should be set."
```

Comment:
in a httprule or in the httprule? which one is correct?

Reply:
addressed